### PR TITLE
Add config defaults to models.yml settings.

### PR DESCRIPTION
**Pull request by reiterl**
Mostly add the config defaults to the models.yml settings.

Exclude special cases:
conference_auto_connect_next_speakers
user_pdf_url
user_email_body

This is the most setting defaults being set, but some are missing.

### CODE REVIEW

File: docs/models.yml [hunk 1]
```diff
@@ -458,35 +520,62 @@ meeting:
...
+    type: string
+    default: Motions
   motions_export_preamble: string
-  motions_export_submitter_recommendation: boolean
-  motions_export_follow_recommendation: boolean
+  motions_export_submitter_recommendation:
+    type: boolean
+    default: False
+  motions_export_follow_recommendation:
+    default: boolean
```
Comment (FinnStutzenstein) [inline docs/models.yml:578]: this is not right. The default is false, the type is boolean

**Review by FinnStutzenstein — COMMENTED** [review]
Some more defaults:
- projector_default_countdown_time: 60
- projector_countdown_warning_time: 0
- conference_auto_connect_next_speakers: This is actually an integer (changed recently) with default 0
- list_of_speakers_amount_next_on_projector: should be an integer with default -1
- list_of_speakers_show_amount_of_speakers_on_slide: True
- motions_number_type: per_category
- motion_poll_default_type, assignment_poll_default_type, poll_default_type: analog
- *default_100_percent_base: YNA
- users_allow_self_set_present: False

**Comment by FinnStutzenstein** [comment]
@reiterl Ping

**Comment by reiterl** [comment]
I have added the missing setting-default, fix the "default: boolean" bug and rebased it.

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**